### ISSUE TITLE
Tome of Alkahest does not lose durability in HermitPack v1.2.1

### ISSUE THREAD

**Issue by fmease** [P3pp3rF1y/Reliquary#184]
Reliquary v1.3.3.581
Forge 12.18.2.2099

When duplicating an item, the durability of the tome stays the same.
When I set the charge manually to circumvent this bug (I am using /entitydata @e[type=Item,r=3] {Item:{tag:{charge:<NEW>}}} just dropping the tome in front of me), the TOME.Item.tag.durability (or what soever) will update after you craft again.
Tested it in survival using gold ingots, nether stars and redstone dust.

**Comment by predakanga** [comment]
Just did some manual bisecting, problem was introduced between 1.3.3.571 and 1.3.3.578

**Comment by P3pp3rF1y** [comment]
Could you attach yor xreliquary config file? I have only changed a bit with loading crafting recipes from config so there must be something there.

**Comment by predakanga** [comment]
Sure thing - had to zip it, GitHub doesn't allow .cfg files: [xreliquary.zip](https://github.com/P3pp3rF1y/Reliquary/files/557906/xreliquary.zip)

It's the default config from "FTB Unstable 1.10"

**Comment by P3pp3rF1y** [comment]
So yeah, as part of the config I have also simplified the key that I use to identify items so that it always includes metadata even for single metadata items.  And apparently I have updated the two places in crafting recipe, but not the third where the charge used is calculated and it couldn't find the item's cost and thus defaulted to 0. It's fixed now. I will see how soon I will be able to get a release out.
